### ISSUE TITLE
RQ2A.210505.002.2021.05.04.01 Beta Test Results

### ISSUE THREAD

**Issue by bill-in-mtl** [GrapheneOS/beta-feedback#12]
Beta Testing Checklist

Device: Sunfish(Pixel 4A)
Build number: RQ2A.210505.002.2021.05.04.01

Receive incoming calls: Works
Make outgoing phone calls: Works
Receive incoming texts: Works
Send outgoing texts: Works

Internet connectivity: Works
Internet browsing: Works
Vanadium use Works
Video playback: Works
Renders websites: Works

Media playback: Works
Bluetooth Audio Pairing: Works
Bluetooth Audio Playback: Works
Bluetooth Audio Calling: Works
PDF Viewer: Working
Alarms/Timer: n/a

Cameras  (Front and Rear): Works,

Handset Microphones: Works
Handset Speakers: Works

Keyboards: Works

File Browsing: Works
Media Transfer to PC: Works
Application Installs: Works
Switch Between Profiles: n/a
Log Out Profiles: n/a
Seedvault Backups: n/a

**Comment by akc3n** [comment]
Date: May 4th, 2021
Device: Sargo (Pixel 3a)
Build number: RQ2A.210505.002.2021.05.04.01

Receive incoming calls: Works
Make outgoing phone calls: Works
Receive incoming SMS: Works
Send outgoing SMS: Works
Receive incoming MMS: Works
Send outgoing MMS: Works
Internet connectivity: Works
Internet browsing: Works
Vanadium use Works

Video playback: Works
Renders websites: Works
Media playback: Works
Bluetooth Audio Pairing: Works
Bluetooth Audio Playback: Works
Bluetooth Audio Calling: Works
PDF Viewer: Working
Alarms: Works
Timer: Works
Cameras (Front and Rear): Works
Video: Works
Handset Microphones: Works
Handset Speakers: Works
USB Devices & Peripherals: N/A

USB Mass Storage: Works
Keyboards: Works
USB Digital Audio Output: Works
File Browsing: Works
Media Transfer to PC: Works
Application Installs: Works
Switch Between Profiles: Works
Log Out Profiles: Works

Owner Profile Seedvault Backups: Works
Owner Profile Seedvault Restore: Works

Secondary User Profile Seedvault Backups: Works
Secondary User Profile Seedvault Restore: Works

**Comment by flawedworld** [comment]
Date: May 5th, 2021
Device: Redfin (Pixel 5)
Build number: RQ2A.210505.003.2021.05.04.01

Receive incoming calls: Works
Make outgoing phone calls: Works
Receive incoming SMS: Works
Send outgoing SMS: Works
Receive incoming MMS: Untested
Send outgoing MMS: Untested
Internet connectivity: Works
Internet browsing: Works
Vanadium use: Works

Video playback: Works
Renders websites: Works
Media playback: Works
Bluetooth Audio Pairing: Works
Bluetooth Audio Playback: Works
Bluetooth Audio Calling: Works
PDF Viewer: Works
Alarms: Works
Timer: Works
Cameras (Front and Rear): Works
Video: Works
Headset Microphones: Untested
Headset Speakers: Untested
USB Devices & Peripherals:

USB Mass Storage: Untested
Keyboards: Works
USB Digital Audio Output: Untested
File Browsing: Works
Media Transfer to PC: Works
Application Installs: Works
Switch Between Profiles: Untested
Log Out Profiles: Untested
Seedvault Backups: Untested

Other Notes:
Keyboard crash bug still a thing.

**Comment by Peter-Easton** [comment]
**Date:** May 4th, 2021
**Device:** Flame (Pixel 4)
**Build number:** RQ2A.210505.002.2021.05.04.01

**Receive incoming calls:** Works
**Make outgoing phone calls:** Works
**Receive incoming SMS:** Works
**Send outgoing SMS:** Works
**Receive incoming MMS:** Works
**Send outgoing MMS:** Works
**Data connectivity:** Works
**Wi-Fi connectivity:** Works
**Internet browsing:** Works
**Vanadium use** Works
 * Video playback: Works
 * Renders websites: Works

**Media playback:** Works
**Bluetooth Audio Pairing:** Works
**Bluetooth Audio Playback:** Works
**Bluetooth Audio Calling:** Works
**PDF Viewer:** Works
**Alarms:** Not Tested
**Timer:** Not Tested
**Cameras (Front and Rear):** Works
**Video:** Works (The bug in selfie video camera on OpenCamera is preexisting and caused by an upstream issue. A workaround is in place.)
**Handset Microphones:** Works
**Handset Speakers:** Works
**USB Devices & Peripherals:**
* USB Mass Storage: Works
* Keyboards: Not Tested
* USB Digital Audio Output: Works 

**File Browsing:** Works
**Media Transfer to PC:** Works
**Application Installs:** Works
**Switch Between Profiles:** Works (The bug in the keyboard not appearing when switching back to the owner profile is preexisting and a suspected problem upstream)
**Log Out Profiles:** Works

**Seedvault Backups:**
 * **Owner Profile**: Works
 * **Secondary Profile**: Works

**Customization:** Works perfectly.

**Other Notes:**
Everything looks good so far. I didn't find anything broken that wasn't broken before. 

I tested SeedVault by restoring apps. It restores apps and the appdata that is permitted to be backed up, as well as restoring the contact list.

Edited: Correction, it DOES restore system settings, it just doesn't restore themes. 

Notifications on the second profile seem to not display properly, but otherwise functionality is unaffected.

**Comment by southwestgit** [comment]
Beta Testing Checklist
Date: May 4, 2021
Device: 3a
Build number: RQ2A.210505.002.2021.05.04.01

Receive incoming calls: Not Tested
Make outgoing phone calls: Yes
Receive incoming texts: Not Tested
Send outgoing texts: Not Tested
Internet connectivity: Yes
Internet browsing: Yes
Vanadium use:
 * Video playback: Yes
 * Renders websites: Yes
Media playback: Yes
PDF Viewer: Not Tested
Alarms/Timer: Yes
Cameras (Front and Rear): Yes
Video: Yes
Microphones: Yes
USB Devices & Peripherals: Not Tested
File Browsing: Not Tested
Media Transfer to PC: Not Tested
Application Installs: Yes
Switch Between Profiles: Yes
Log Out Profiles: Yes

Other Notes:
Limited ability for me to test at current location.

**Comment by Jawshy** [comment]
# Beta Testing Checklist

**Date**: 5 May 2021
**Device**: Bonito (Pixel 3a XL)
**Build number**: RQ2A.210505.002.2021.05.04.01

**Internet Connectivity**: ✅
**Internet Browsing**: ✅
**Vanadium Use**: 
* Video Playback: Not tested
* Renders Websites: ✅

**Media Transfer to PC**: ✅
**Application Installs**: ✅ Via F-Droid
**Switch Between Profiles**: ✅
**Log Out Profiles**: ✅
**Seedvault Backups**: 
* Owner Profile<sup>1</sup>: System apps: SMS text messages✅, device settings✅, call history✅, local contacts✅; installed apps
* Secondary Profile<sup>2</sup>: System apps: SMS text messages (not tested), device settings⚠️<sup>3</sup>, call history (not tested), local contacts✅; installed apps✅

**Other Notes**: 
<sup>1</sup> Only tested backup of owner user; not restore. 
<sup>2</sup> Tested automatic restore where app data was restored upon reinstalling app, and tested restoring user profile upon initial user set up. 
<sup>3</sup> Device settings like dark theme, battery percentage, time format, etc. were not backed up or restored

**Comment by Steven0054** [comment]
Beta Testing Checklist

Device: bramble (Pixel 4a 5G)
Build number: RQ2A.210505.002.2021.05.04.01

Receive incoming calls: Works (VoLTE, Legacy, and VoWiFi)
Make outgoing phone calls: Works (VoLTE, Legacy, and VoWiFi)
Receive incoming texts: Works
Send outgoing texts: Works
Receive incoming MMS: Works
Send outgoing MMS: Works
Internet connectivity: Works
Internet browsing: Works
Vanadium use:
- Video playback: Works (YouTube, Dailymotion, Viprow)
- Renders websites: Works (YouTube, Dailymotion, Viprow, duckduckgo, amazon, https://wiki.codeaurora.org/xwiki/bin/QAEP/release)

Media playback: Works (mp4/mkv/aac/flac)
PDF Viewer: Works (3 documents)
Renders websites: Works
Bluetooth Audio Pairing: Works
Bluetooth Audio Playback: Works
Bluetooth Audio Calling: Works
NFC: Works
PDF Viewer: Works
Alarms: Works
Timer: Works
Cameras (Front and Rear): Works
Video: Works
Handset Microphones: Works
Handset Speakers: Works
USB Devices & Peripherals: Works
USB Mass Storage: Works
Keyboards: Works
USB Digital Audio Output: Works

File Browsing: Works
Media Transfer to PC: Works
Application Installs: Works
Switch Between Profiles: Works
Log Out Profiles: Works

Owner Profile Seedvault Backups: n/a
Owner Profile Seedvault Restore: n/a

Secondary User Profile Seedvault Backups: Works
Secondary User Profile Seedvault Restore: Works

**Comment by fomijafi** [comment]
Date: May 5th, 2021
Device: Blueline (Pixel 3)
Build number: RQ2A.210505.002.2021.05.04.01

Receive incoming calls: Works
Make outgoing phone calls: Works
Receive incoming SMS: Works
Send outgoing SMS: Works
Receive incoming MMS: Works
Send outgoing MMS: Works
Data connectivity: Works
Wi-Fi connectivity: Works
Internet browsing: Works
Vanadium use Works

Video playback: Works
Renders websites: Works
Media playback: Works
Bluetooth Audio Pairing: Works
Bluetooth Audio Playback: Works
Bluetooth Audio Calling: Works
PDF Viewer: Works
Alarms: Work
Timer: Work
Cameras (Front and Rear): Works
Video: Works
Handset Microphones: Works
Handset Speakers: Works
USB Devices & Peripherals:

USB Mass Storage: Not tested
Keyboards: Not Tested
USB Digital Audio Output: Works
File Browsing: Works
Media Transfer to PC: Works
Application Installs: Works
Switch Between Profiles: Wors
Log Out Profiles: Works

Seedvault Backups:

Owner Profile: Not tested
Secondary Profile: Works
Customization: Works

Other Notes:

I made a seedvault backup to nextcloud of a secondary profile. I then made a new profile and clicked Install Nextcloud option in setup. Nothing happened. Skipping past the setup I was unable to restore from backup (I know this is as intended)

So the backup worked but I didn't get to restore it. This could be my own ignorance about seedvault as I've never used it bedfore.
